component/src/main/java/org/wso2/extension/siddhi/io/ibmmq/source/IBMMQSource.java

```diff
@@ -0,0 +1,222 @@
+                        description = "Host address of the IBM MQ server",
+                        type = DataType.STRING),
+                @Parameter(name = IBMMQConstants.PORT,
+                        description = "Port of the IBM MQ server",
+                        type = DataType.STRING),
+                @Parameter(name = IBMMQConstants.CHANNEL,
+                        description = "Channel used to connect to the MQ server",
+                        type = DataType.STRING),
+                @Parameter(name = IBMMQConstants.QUEUE_MANAGER_NAME,
+                        description = "Name of the Queue Manager",
+                        type = DataType.STRING),
+                @Parameter(name = IBMMQConstants.USER_NAME,
+                        description = "User name of the server. If this is not provided, " +
+                                "will try to connect without both username and password",
+                        type = DataType.STRING,
+                        optional = true,
+                        defaultValue = "null"),
+                @Parameter(name = IBMMQConstants.PASSWORD,
+                        description = "Password of the server. If this is not provided, will try to connect without " +
+                                "both username and password",
+                        type = DataType.STRING,
+                        optional = true,
+                        defaultValue = "null"),
+        },
+        examples = {
+                @Example(description = "This example shows how to connect to an IBM message queue and "
+                        + "receive messages.",
+                        syntax = "@source(type='ibmmq',"
+                                + "destination.name='Queue1',"
+                                + "host='192.168.56.3',"
+                                + "port='1414',"
+                                + "channel='Channel1',"
+                                + "queue.manager = 'ESBQManager',"
+                                + "password='1920',"
+                                + "username='mqm',"
+                                + "@map(type='text'))"
+                                + "define stream SweetProductionStream(name string, amount double);"),
+        }
+)
+
+public class IBMMQSource extends Source {
+    private static final Logger LOG = LoggerFactory.getLogger(IBMMQSource.class);
+    private SourceEventListener sourceEventListener;
+    private OptionHolder optionHolder;
+    private MQConnection connection;
+    private MQQueueConnectionFactory connectionFactory;
+    private Session session;
+    private Queue queue;
+    private MessageConsumer consumer;
+    private IBMMessageConsumer ibmMessageConsumer;
+    private ScheduledExecutorService scheduledExecutorService;
+    private String userName;
+    private String password;
+    private String queueName;
+    private SiddhiAppContext siddhiAppContext;
+    private boolean isSecured = false;
+
+
```

Comment:
Please remove unnecessary newlines

Reply:
This is fixed by 6e40355446fd7c544b8b0332258d5ca6ba556a0f